### ISSUE TITLE
in pyFai-calib2

### ISSUE THREAD

**Issue by raph38130** [silx-kit/pyFAI#1287]
```ERROR:pyFAI-calib2.UNCAUGHT:pixelToData() got an unexpected keyword argument 'check'
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/pyFAI/gui/tasks/GeometryTask.py", line 279, in eventFilter
    coord = widget.pixelToData(pos.x(), pos.y(), axis="left", check=False)
TypeError: pixelToData() got an unexpected keyword argument 'check'
INFO:pyFAI.massif:Image size is (1024, 2048)
INFO:pyFAI.massif:Binning size is [1, 2]
INFO:pyFAI.massif:Labeling found 8457 massifs.
INFO:pyFAI.gui.helper.RingExtractor:Extracting datapoint for ring 9 (2theta = 0.30 deg); searching for 361 pts out of 3035 with I>484.3, dmin=0.7
INFO:pyFAI.gui.helper.RingExtractor:Extracting datapoint for ring 10 (2theta = 0.31 deg); searching for 361 pts out of 3674 with I>283.6, dmin=0.8
WARNING:pyFAI.calibrant:Forbidden to change the wavelength once it is fixed !!!!
WARNING:pyFAI.calibrant:6.573478410740104e-13 != 4.132806579880239e-11, delta= -4.067071795772838e-11
WARNING:pyFAI.calibrant:Forbidden to change the wavelength once it is fixed !!!!
WARNING:pyFAI.calibrant:6.573478410740104e-13 != 4.132806579880239e-11, delta= -4.067071795772838e-11
```

**Comment by vallsv** [comment]
I think it depend of the version of `silx` you use.

Do you know which version you use?

**Comment by vallsv** [comment]
It is supposed to be `silx >= 0.10`, but maybe it have to be `>= 0.11`

**Comment by kif** [comment]
Apparently the bug is in pyFAI ... there is a quick-fix in the PR for 0.19
